monkestation/code/datums/traits/negative.dm

```diff
@@ -63,3 +63,15 @@
 /datum/quirk/kleptomaniac/remove()
 	var/mob/living/carbon/human/H = quirk_holder
 	H.cure_trauma_type(/datum/brain_trauma/mild/kleptomania, TRAUMA_RESILIENCE_ABSOLUTE)
+
+/datum/quirk/gigantism
+	name = "Gigantism"
+	desc = "You're huge! You start the round with an immutable gigantism mutation. Even works on species without DNA!"
+	value = -4
```

Comment:
4 trait points seems way too much for a genetic mutation that you can go get cured with zero effort.
I'd say making this a neutral quirk for 0 would be acceptable.

Reply:
It works like wizard mutations and isn't removable in such.